client/post-editor/editor-ground-control/index.jsx

```diff
@@ -91,10 +93,23 @@ export default React.createClass( {
 			showDateTooltip: false,
 			firstDayOfTheMonth: this.getFirstDayOfTheMonth(),
 			lastDayOfTheMonth: this.getLastDayOfTheMonth(),
-			needsVerification: config( 'email_verification_gate' ) && userUtils.needsVerificationForSite( this.props.site ),
+			needsVerification: config( 'email_verification_gate' ) && this.props.userUtils.needsVerificationForSite( this.props.site ),
 		};
 	},
 
+	componentWillReceiveProps: function( nextProps ) {
+		this.setState( {
```

Comment:
Why do we store this in state at all? Seems to fall within the same anti-pattern described in [_Props in getInitialState Is an Anti-Pattern_](https://facebook.github.io/react/tips/props-in-getInitialState-as-anti-pattern.html).